modules/social_features/social_activity/modules/social_activity_filter/social_activity_filter.info.yml

```diff
@@ -0,0 +1,6 @@
+name: 'Social Activity Filter'
+description: 'Provides filtered activity stream for Open Social.'
+type: module
+core: 8.x
+package: Social
```

Comment:
@volodymyr-sydor-lemberg I think it's better to set package `Social (experimental)`

Reply:
done